src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -46,13 +46,12 @@ object Homework :
       else true
 
     def and(left: Boolean, right: Boolean): Boolean =

```

Comment:
You need to implement it like it works in Java, means that if `left` is false then we don't need to calculate right.
Hint: call-by-name strategy.
Also add test for this case.

Reply:
Ok, now I will add tests to all tasks.